PyDSS/NetworkModifier.py

```diff
@@ -18,10 +18,7 @@ class Modifier():
      }
 
     def __init__(self, dss, run_command, SimulationSettings):
-        if SimulationSettings["Logging"]["Pre-configured logging"]:
-            LoggerTag = __name__
-        else:
-            LoggerTag = getLoggerTag(SimulationSettings)
```

Comment:
Is there a reasonyou are removing  the pre-configured logger?

Reply:
It's not needed anymore. I think that I added it for DISCO at some point but then no longer needed it. It is not being used in this branch. Are you using it in master?

Reply:
To be more clear I was previously only using this functionality when calling `PyDssProject.run()` from outside of PyDSS where logging was already configured. I removed that functionality in this branch. Now, logging is always configured within PyDSS. Would you prefer the previous functionality?

Reply:
I am fine with this implementation